conf/app.routes

```diff
@@ -6,6 +6,8 @@ POST          /new-form/:formTemplateId/:userId/:affinityGroup
 PUT           /forms/:userId/:formTemplateId                                                            uk.gov.hmrc.gform.form.FormController.updateFormDataPlain(userId: UserId, formTemplateId: String)
 PUT           /forms/:userId/:formTemplateId/:accessCode                                                uk.gov.hmrc.gform.form.FormController.updateFormData(userId: UserId, formTemplateId: String, accessCode: AccessCode)
 
+POST          /forms/:userId/:formTemplateId/from-legacy                                                uk.gov.hmrc.gform.form.FormController.createFormFromLegacy(userId: UserId, formTemplateId: String)
```

Comment:
Let's use `FormTemplateId` type instead of `String`. (Other places where `String` is used for `formTemplateId`, are probably overlooks)

Reply:
There is not endpoint for 
```
s"$baseUrl/forms/${userId.value}/${formTemplateId.value}/${accessCode.value}/from-legacy"
```
Not we need to ask if agent support is needed or not for this functionality.